jhove-modules/pdf-hul/src/main/resources/edu/harvard/hul/ois/jhove/module/pdf/ErrorMessages.properties

```diff
@@ -153,3 +153,6 @@ PDF-HUL-147 = Page tree node not found.
 PDF-HUL-148 = PDF minor version number is greater than 7.
 PDF-HUL-149 = Invalid indirect destination - referenced object ''{0}'' cannot be found
 PDF-HUL-150 = Cross-reference stream must be a stream
+PDF_HUL_151 = Unexpected error occurred while attempting to read the cross-reference table
+PDF_HUL_152 = The mandatory OE is missing
+PDF_HUL_153 = The mandatory UE is missing
```

Comment:
To add a little more detail and bring them in line with other similar PDF messages:
"Encrypt dictionary has no OE key or it has a null value"
"Encrypt dictionary has no UE key or it has a null value"